### DISCUSSION TITLE
Open using Rosetta missing in Kitty.app

### DISCUSSION THREAD

**Discussion by iamajvillalobos** [kovidgoyal/kitty#5914] [Q&A] [answered]
Is it possible to have a rosetta option in Kitty?

Terminal|Kity
---------|--------
<img src="https://user-images.githubusercontent.com/7567758/213870478-c8b0339e-7f12-4b53-8a22-6974e1e9abfa.png" width=300 /> |  <img src="https://user-images.githubusercontent.com/7567758/213870481-4b96625b-64c3-49de-88d3-a99ea76415ec.png" width=300 />

**Comment by kovidgoyal** [comment]
Why?

**Reply by iamajvillalobos** [reply]
I have an old rails project that I need to run in i386 arch so I that I can install libraries that don't have support yet for m1 silicons

**Comment by kovidgoyal** [comment, marked as answer]
On Sat, Jan 21, 2023 at 09:35:42AM -0800, AJ Villalobos wrote:
> I have an old rails project that I need to run in i386 arch so I that I can install libraries that don't have support yet for m1 silicons

You can simply use the arch command to start a kitty instance using the
x86_64 binaries since kitty is a universal binary.
